packages/api-client/src/components/AddressBar/AddressBar.vue

```diff
@@ -157,7 +157,7 @@ onBeforeUnmount(() => events.hotKeys.off(handleHotKey))
               :isEditable="!isReadOnly"
               isSquare
               :method="activeRequest.method"
-              teleport=".scalar-client"
+              teleport
```

Comment:
should we maintain a `scalar-client` class here to keep the style of http methods?

Reply:
Yep! Fixed 🫡